featurebyte/models/precomputed_lookup_feature_table.py

```diff
@@ -83,28 +83,32 @@ def get_precomputed_lookup_feature_table_name(
     return f"{source_feature_table_name}_via_{serving_names_suffix}_{unique_identifier}"
 
 
-def get_precomputed_lookup_feature_tables(
+def get_precomputed_lookup_feature_table(
```

Comment:
@gee-senbong I made some changes to this method so that it operates on a specific deployment, similar to the workflow in registry construction. This is so that `deployment_ids` in feature tables can be tracked correctly.

Reply:
If i understood it, it means the precomputed lookup is deployment specific and it can be shared between feature lists, right?

Reply:
The same precomputed lookup feature table can be shared between multiple deployments / feature lists. The `deployment_ids` of the feature table tracks the deployments that are using it.

Reply:
I think i got it now. I think this one is cleaner as we only need to update based on the triggering feature list rather than using all affected feature lists.